src/models/analytics-panel.ts

```diff
@@ -59,6 +59,7 @@ const schema = Joi.object().keys({
   layout: layoutSchema.default([]),
   cards: Joi.array().items(Joi.object().keys({
     title: Joi.string().required().example('My widget'),
+    subTitle: Joi.string().allow('').optional(),
```

Comment:
* Shall we write it as `subtitle`? Wikipedia writes it as one word: https://en.wikipedia.org/wiki/Subtitles
* `.optional()` is not needed: if you do not specify `.required()` it is optional by default

Reply:
No idea what I was thinking there with the camelcase :)
Removed optional() everywhere